resources/public/api/conf/3.0/list.yaml

```
@@ -24,7 +24,11 @@ get:
             <tr>
                 <td><p>NOT_FOUND</p></td>
                 <td><p>Simulates the scenario where no data is found.</p></td>
-            </tr>                       
+            </tr>   
+             <tr>
+                <td><p>DYNAMIC</p></td>
+                <td><p>The following response parameters will be based on the data submitted in the request:<ol><li>taxYear</li><li>calculationTimestamp</li><li>finalDeclarationTimestamp</li></ol></p></td>
```

can make it more readable? and we've been using <ul> instead of <ol>. Do we want a numbered list or bullet points?
```suggestion
            <td><p>The following response values will change to correspond to the values submitted in the request:
              <ul>
                  <li>taxYear</li>
                  <li>calculationTimestamp</li>
                  <li>finalDeclarationTimestamp</li>
              </ul>
            </p></td>
```